docusaurus/docs/iOS/uikit/custom-components.md

```diff
@@ -153,7 +153,7 @@ class iMessageChannelListItemView: ChatChannelListItemView {
 
     override func setUpAppearance() {
         super.setUpAppearance()
-`
+
         customUnreadView.backgroundColor = tintColor
         customUnreadView.layer.masksToBounds = true
         customUnreadView.layer.cornerRadius = 5
```

Comment:
Most likely this is also wrong, cornerRadius should not be applied in `setUpAppearance` :/

Reply:
@nuno-vieira, it comes from [here](https://github.com/GetStream/stream-chat-swift/blob/develop/Examples/iMessageClone/iMessageChatChannelListItemView.swift#L16). By the way, I don't think we need `cornerRadius` on iMessageClone, do we?

Reply:
Updated

Reply:
Oh, just checked it across all docs, actually, it is everywhere. Let me commit one more time to fix it there.

Reply:
@nuno-vieira, done, check it out please 👍